db/migrate/20150618112012_add_old_route_to_tags.rb

```diff
@@ -0,0 +1,10 @@
+class AddOldRouteToTags < ActiveRecord::Migration
+  def up
+    add_column :tags, :has_legacy_route_without_namespace, :boolean
```

Comment:
I think this would be better disallowing NULL and with a default of false, to avoid a tri-state boolean.